mne/epochs.py

```diff
@@ -1720,6 +1724,10 @@ class EpochsArray(_BaseEpochs):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
         Defaults to raw.verbose.
+
+    See Also
+    --------
+    io.RawArray, mne.EvokedArray
```

Comment:
Either `mne.io.RawArray` and `mne.EvokedArray` or `io.RawArray` and `EvokedArray`. Hopefully the latter will work fine